taxi/forms.py

```diff
@@ -0,0 +1,59 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False
+    )
+
+    class Meta:
+        model = Car
+        fields = "__all__"
+
+
+class DriverForm(UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = (
+            "username",
+            "password1",
+            "password2",
+            "license_number",
+            "email",
+            "first_name",
+            "last_name"
+        )
```

Comment:
why don't u select fields like this? `fields = UserCreationForm.Meta.fields + ("custom_field",)`

Reply:
do we need some of the default fields? i wanted to get rid of some of the unnecessary ones, such as superuser status, permissions, etc

Reply:
Also you can provide fields in wanted ordere, while using meta they always will be in meta order.